README.md

```diff
@@ -76,3 +76,25 @@ Run the following to ensure the MPACT compiler builds and runs correctly.
 ```shell
 cmake --build build --target check-mpact
 ```
+
+
+## Notes
+
+1. Shells other than bash/sh/etc. require a different `activate` script, as shown.  Because the python environment has to be set up for every session, we recommend putting it in your .*sh startup file.
```

Comment:
```suggestion
1. Shells other than bash/sh/etc. require a different `activate` script, as shown. Because the python environment has to be set up for every session, we recommend putting it in your .*sh startup file.
```